pkgs/development/python-modules/dijitso/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv
+, lib
+, fetchurl
+, python3Packages
+, dolfin
+}:
+
+python3Packages.buildPythonPackage rec {
+  pname = "dijitso";
+  inherit (dolfin) version;
+
+  src = fetchurl {
+    url = "https://bitbucket.org/fenics-project/dijitso/downloads/dijitso-${version}.tar.gz";
+    sha256 = "1ncgbr0bn5cvv16f13g722a0ipw6p9y6p4iasxjziwsp8kn5x97a";
+  };
+
+  propagatedBuildInputs = with python3Packages; [ numpy six ];
+  checkInputs = with python3Packages; [ pytest ];
+
+  preCheck = ''
+    export HOME=$PWD
+  '';
+
+  checkPhase = ''
+    runHook preCheck
+    py.test test/
+    runHook postCheck
+  '';
+
+  meta = {
+    description = "Distributed just-in-time shared library building";
+    homepage = "https://fenicsproject.org/";
+    platforms = lib.platforms.all;
+    license = lib.licenses.lgpl3;
```

Comment:
Please add yourself as a maintainer when adding a new package.

https://nixos.org/manual/nixpkgs/stable/#reviewing-contributions-new-packages

Reply:
I'm not sure if I should add myself, because this package originally had no maintainers, and I don't think I am qualified enough to handle maintenance of this package. I made this PR because I need the dolfin library for another PR I have open, so I split these modules apart. I have no real idea how they fit together.